Example pipelines/glm_pipeline.m

```diff
@@ -0,0 +1,112 @@
+%% GLM analysis in Homer3 toolbox
+% This livescript will demonstrate how to perform GLM analysis using Homer3 
+% toolbox. The following analysis steps will be performed:
+%% 
+% * <#H_18EE30F8 Read in data>
+% * <#H_0260A343 Convert raw intensity data to optical densities>
+% * <#H_E289EE87 Bandpass filter>
+% * <#H_24060CF9 Convert optical densities to hemoglobin concentration changes>
+% * <#H_DF21F6F5 Apply GLM>
+% Read in data
+% First, the data will be loaded from <https://fnirs.org/resources/software/snirf/ 
+% Shared Near Infrared Spectroscopy Format> (SNIRF), which was designed to facilitate 
+% sharing and analysing NIRS data. A detailed description of the format can be 
+% found <https://github.com/fNIRS/snirf/blob/master/snirf_specification.md#snirf-data-format-summary 
+% here>. We will first select a measurement file. 
+
+clear all; close all; clc
+[fname, pname] = uigetfile('*.snirf', 'Select snirf file(s)', 'MultiSelect', 'on');
+
+% load raw intensity signals and metadata into data_nirs 
+if iscell(fname) 
```

Comment:
I believe converting `fname` to a String type would allow avoiding this if/else class branching, i.e. same code pathway for scalar and vector cases.